monkeys/src/commonMain/kotlin/com/wire/kalium/monkeys/importer/TestData.kt

```diff
@@ -17,43 +17,91 @@
  */
 package com.wire.kalium.monkeys.importer
 
+import com.wire.kalium.logic.data.conversation.ConversationOptions
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
 data class TestDataJsonModel(
-    @SerialName("backends")
-    val backends: List<BackendDataJsonModel>
+    @SerialName("testCases") val testCases: List<TestCase>,
+    @SerialName("backends") val backends: List<BackendDataJsonModel>
 )
 
+@Serializable
+data class TestCase(
+    @SerialName("name") val name: String,
+    @SerialName("conversationDistribution") val conversationDistribution: Map<String, GroupConfig>,
+    @SerialName("setup") val setup: List<Action> = listOf(),
+    @SerialName("actions") val actions: List<Action>
+)
+
+@Serializable
+sealed class UserCount {
+    @Serializable
+    @SerialName("PERCENTAGE")
+    data class Percentage(@SerialName("value") val value: UInt) : UserCount()
+
+    @Serializable
+    @SerialName("FIXED_COUNT")
+    data class FixedCount(@SerialName("value") val value: UInt) : UserCount()
+}
```

Comment:
Just as a heads up, this will require an implicit `type` field in order for Kotlinx-serialization to select which one is being deserialized.

```json
"userCount": {
    "type": "PERCENTAGE",
    "value": 48
}
```

```json
"userCount": {
    "type": "FIXED_COUNT",
    "value": 1000
}
```

I like it :)